From 09fe6ebe02d439c507c0bad6e500c9212f56e8e3 Mon Sep 17 00:00:00 2001
From: Jadefalke <79963380+Jadefalke256@users.noreply.github.com>
Date: Wed, 3 May 2023 16:30:43 +0200
Subject: [PATCH] Remove == bashism in favor of posix compliancy

Remove `==` bashism. This leads to build-fails with other posix-compliant shells such as `dash` and does not affect `bash` itself.
--- a/libraries/terminfo/configure.ac
+++ b/libraries/terminfo/configure.ac
@@ -18,7 +18,7 @@ if test "x$curses_libraries" != "xNONE"; then
   TERMINFO_LIB_DIRS=$curses_libraries
 fi
 
-if test "x$curses_library" == "xNONE"; then
+if test "x$curses_library" = "xNONE"; then
 AC_PROG_CC()
 
 AC_CHECK_LIB(tinfow, setupterm, HaveLibCurses=YES; LibCurses=tinfow,
